app/models/concerns/filters/projects_filter.rb

```diff
@@ -0,0 +1,14 @@
+module Filters
+  module ProjectsFilter
```

Comment:
this is used in only one place, I don't see need for this

Reply:
Thanks for the input!